static_report/src/utils/icons.tsx

```diff
@@ -0,0 +1,19 @@
+import { IconType } from 'react-icons/lib';
```

Comment:
You can put this code to `static_report/src/components/Icons/index.tsx`

Reply:
Moved